clients/admin-ui/src/features/data-discovery-and-detection/action-center/tables/DiscoveredSystemAggregateTable.tsx

```diff
@@ -50,20 +50,38 @@ export const DiscoveredSystemAggregateTable = ({
 
     // Loading states
     anyBulkActionIsLoading,
+
+    // Okta-specific functionality
+    isOktaApp,
+    oktaActiveTab,
+    setOktaActiveTab,
+    oktaFilterTabs,
+    oktaFilterCounts,
   } = useDiscoveredSystemAggregateTable({ monitorId });
 
   return (
     <>
       <Menu
         aria-label="Asset state filter"
         mode="horizontal"
-        items={filterTabs.map((tab) => ({
-          key: tab.hash,
-          label: tab.label,
-        }))}
-        selectedKeys={[activeTab]}
+        items={
+          isOktaApp
+            ? oktaFilterTabs.map((tab) => ({
+                key: tab.value,
+                label: `${tab.label} (${oktaFilterCounts[tab.value] || 0})`,
+              }))
+            : filterTabs.map((tab) => ({
+                key: tab.hash,
+                label: tab.label,
+              }))
+        }
+        selectedKeys={[isOktaApp ? oktaActiveTab : activeTab]}
```

Comment:
same here, if it is okta, then hook above should return correct activeTab, not a new var.